### PR TITLE
fix(OnyxSelect): fix cursor always jumping to the end of the search input

### LINKED ISSUE #1466: Fix OnyxSelect search input behavior
1. Open [this example](https://playground.onyx.schwarz/#eNp9kk1vwjAMhv9KlPNWDtuJwaRt4rAJDTR2W6YpFANhaRIlLh9C/Pc5aUs5dKhSZedx3zqvfeRPzmXbEnifD0LulUOmpVkNBccgOAuApXsURhXOemRHNjGH/Qw05MhObOltwQQPCm8tnQv+IIwwuTUBmXWoKGBD9iUMY8f4YlS8lboEwfsUSuc0hTcN0nIOukLUVUSRnBLv+n4uDT3dAs81axT+0wjo5W4O3h+6dWYXPGkJ8023HPQqs8gaShAKpyUCZYwNWosqvaRGho72ksqAPE0syRFVAcd1xZjiqryh/dpGgnXUEPphDmurF+AJTtssy7KmZqdwPQPp83XMe9TeoHfuld/QiGlWS7XKNsEa2oBkkOC5LZzS4CfNH/utdVJru3tLZ+hLOFuWryH/7TjfBFqLaOTUQwC/vZw3Sr8CMiLi0ewd9hSfYWEXZVyBK/ADgtVl7LGeeWkW1PZFXer2Ne2uMqvPMNojmNBcKjbabhitRQkvV67etnuX3de7cCIXf7bgoyYZGI0NyE9/eH0W3w==)
2. open the select
3. type in ABC
4. move the cursor BEFORE A
5. Type in DE
=> expected result: "DEABC"
=> actual result:
Image

Somehow, the cursor jumps to the end of the string after typing "D". this also happens when deleting part of the search value.

OnyxMiniSearch itself is not a problem, it only happens inside the select

### PR DESCRIPTION
**Pull request by JoCa96**
Closes #1466

The issue was caused by setting the input value twice: Once in the headless and once via the v-model in the `OnyxMiniSearch`.
To address this, we removed the `inputValue` from the `createComboBox` headless util, as it was only passed through anyway.

### CODE REVIEW

File: packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue [hunk 1]
```diff
@@ -296,6 +292,7 @@ const selectInputProps = computed(() => {
         <OnyxMiniSearch
           v-if="props.withSearch"
           ref="miniSearch"
+          v-model="searchTerm"
```
Comment (larsrickert) [inline packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue:295]: Did you remove the onAutocomplete intentionally?

**Review by larsrickert — CHANGES_REQUESTED** [review]

File: packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue (hunk 1 above)
Comment (JoCa96) [reply packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue]: Yes, but I found a better solution. Please recheck :)

**Review by larsrickert — APPROVED** [review]

File: packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue [hunk 2]
```diff
@@ -293,9 +292,11 @@ const selectInputProps = computed(() => {
       :aria-busy="props.loading"
     >
       <div v-scroll-end class="onyx-select__wrapper" tabindex="-1">
+        <!-- model-value is set here, as it is written be the onAutocomplete callback -->
```
Comment (BoppLi) [inline packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue:295]: Do you mean
```suggestion
        <!-- model-value is set here, as it is written by the onAutocomplete callback -->
```

File: .changeset/mean-rabbits-learn.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+"@sit-onyx/headless": major
+"sit-onyx": patch
+---
+
+fix(OnyxSelect): fix cursor always jumping to the end of the search input
+feat(createComboBox): remove required, but redundant inputValue prop
```
Comment (BoppLi) [inline .changeset/mean-rabbits-learn.md:7]: isn't it rather
```suggestion
refactor(createComboBox): remove required, but redundant inputValue prop
```
? :)

**Pull request merged**